tests/test_unumpy.py

```diff
@@ -179,7 +162,7 @@ def test_wrap_array_func():
     # Function that works with numbers with uncertainties in mat (if
     # mat is an uncertainties.unumpy.matrix):
     def f_unc(mat, *args, **kwargs):
-        return mat.I + args[0] * kwargs["factor"]
+        return numpy.matrix(mat).I + args[0] * kwargs["factor"]
```

Comment:
This function needs to cast to `numpy.matrix` internally. This is because `to_uarray_func` currently drops compatibility with `numpy.matrix`.